spark/src/main/scala/org/apache/spark/sql/delta/ConflictChecker.scala

```diff
@@ -547,6 +548,36 @@ private[delta] class ConflictChecker(
     currentTransactionInfo = currentTransactionInfo.copy(actions = newActions)
   }
 
+  /**
+   * Adjust the current transaction's commit timestamp to account for the winning
+   * transaction's commit timestamp. If this transaction newly enabled ICT, also update
+   * the table properties to reflect the adjusted enablement version and timestamp.
+   */
+  private def resolveTimestampOrderingConflicts(): Unit = {
+    if (!DeltaConfigs.IN_COMMIT_TIMESTAMPS_ENABLED.fromMetaData(currentTransactionInfo.metadata)) {
+      return
+    }
+
+    val winningCommitTimestamp =
+      if (InCommitTimestampUtils.didCurrentTransactionEnableICT(currentTransactionInfo)) {
```

Comment:
I don't see a definition for `InCommitTimestampUtils`... perhaps a file was not added to the PR?

Reply:
Yes, just added it.